### PR TITLE
feat: Implement: Link to Registry, Member, Tokenomics

### PR DESCRIPTION
**Pull request by mohandast52**
- Build changes

S2

SC3M

S77

- Launch changes

S55

- Footer changes

SOO

### CODE REVIEW

File: components/BuildPage/Content.jsx [hunk 1]
```diff
@@ -5,114 +5,169 @@ import { H1 } from '../ui/typography';
...
+          >
+            Tokenomics ↗
+          </a>
+          .
+        </p>
+      </div>
+    </div>
+  </div>
+);
+
```
Comment (truemiller) [inline components/BuildPage/Content.jsx:50]:
```suggestion
const TokenomicsAlert = () => (
  <div
    className="bg-blue-100 rounded-md border-1.5 border-[#91CAFF] text-[#002C8C] p-2 flex"
    role="alert"
  >    
      <div className="py-1 mr-2">
        <InfoIcon />
      </div>
      <div>
        <p>
          You can check available dev rewards for existing/minted agents and
          components on
          {' '}
          <a
            href="https://tokenomics.olas.network/dev-incentives"
            className="text-purple-600"
          >
            Tokenomics ↗
          </a>
          .
        </p>
      </div>
  </div>
);

```

File: components/LaunchPage/Hero.jsx [hunk 2]
```diff
@@ -21,16 +19,27 @@ const Hero = () => (
...
-          </Link>
-        </Button>
+        <div className="flex gap-6">
+          <Button
+            variant="default"
+            size="xl"
+            asChild
+            className="mb-6 w-full md:w-auto"
+          >
+            <Link href="https://launch.olas.network/">Get started</Link>
```
Comment (truemiller) [inline components/LaunchPage/Hero.jsx:29]:
```suggestion
            <ahref="https://launch.olas.network/">Get started</a>
```

File: components/LaunchPage/Hero.jsx [hunk 3]
```diff
@@ -21,16 +19,27 @@ const Hero = () => (
...
+
+          <Button
+            variant="ghostPrimary"
+            size="xl"
+            asChild
+            className="mb-6 w-full md:w-auto"
+          >
+            <Link href="https://registry.olas.network/ethereum/services">
+              Explore launched services
+            </Link>
```
Comment (truemiller) [inline components/LaunchPage/Hero.jsx:40]:
```suggestion
            <a href="https://registry.olas.network/ethereum/services">
              Explore launched services
            </a>
```

**Review by truemiller — DISMISSED** [review]
looks good, few places you could reduce dom depth or change tags

**Review by oaksprout — APPROVED** [review]

**Review by truemiller — APPROVED** [review]

File: components/BuildPage/Content.jsx [hunk 4]
```diff
@@ -5,114 +5,165 @@ import { H1 } from '../ui/typography';
...
+    <div>
+      <p>
+        You can check available dev rewards for existing/minted agents and
+        components on
+        {' '}
+        <a
+          href="https://tokenomics.olas.network/dev-incentives"
+          className="text-purple-600"
+        >
+          Tokenomics ↗
```
Comment (Tanya-atatakai) [inline components/BuildPage/Content.jsx:40]: does it make sense to remove an arrow if we don't use `target="_blank"`?

**Review by Tanya-atatakai — APPROVED** [review]
lgtm, small comment

**Pull request merged**